src/components/Article.jsx

```diff
@@ -1 +1,33 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const dateISO = date.toISOString()
+    .slice(0, 10);
```

Comment:
you can either fit it in one line or put all calls on the same level, not both
```suggestion
  const dateISO = date.toISOString().slice(0, 10);
```

or

```suggestion
  const dateISO = date
    .toISOString()
    .slice(0, 10);
```